ui/components/multichain/multichain-connected-site-menu/multichain-connected-site-menu.stories.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {
+  BackgroundColor,
+  Color,
+} from '../../../helpers/constants/design-system';
+import { MultichainConnectedSiteMenu } from './multichain-connected-site-menu';
+
+export default {
+  title: 'Components/Multichain/MultichainConnectedSiteMenu',
+  component: MultichainConnectedSiteMenu,
+  argTypes: {
+    globalMenuColor: {
+      control: 'text',
+    },
+    text: {
+      control: 'text',
+    },
+    status: {
+      control: 'text',
+    },
+  },
+  args: {
+    globalMenuColor: Color.iconAlternative,
+    status: 'STATUS_NOT_CONNECTED',
```

Comment:
Should import constants here too

Reply:
Yesss [90affbd](https://github.com/MetaMask/metamask-extension/pull/18167/commits/90affbd1e63dec216a1c7edd2445c6839c584794)